iron-overlay-manager.html

```diff
@@ -385,6 +386,7 @@
      * @param {!Element} overlay1
      * @param {!Element} overlay2
      * @return {boolean}
+     * @suppress {missingProperties}
```

Comment:
Hehe yeah I guessed this is still creating problems, was interested to know more context, especially because I added that cast to specifically target that same error noticed in `source: hooli` 😁
May I ask you to remove the cast from this method please? 🙏